.github/workflows/ai_explain.yaml

```diff
@@ -6,7 +6,7 @@ permissions:
 
 on:
   pull_request:
-    types: [opened, synchronize]
+    types: [ready_for_review]
```

Comment:
I would include at least some additional options here here, as your workflow will run only when the PR goes from draft mode into ready for review.